sklearn/datasets/tests/test_base.py

```diff
@@ -188,7 +227,14 @@ def test_loader(loader_func, data_shape, target_shape, n_target, has_descr, file
     if has_descr:
         assert bunch.DESCR
     if filenames:
```

Comment:
If we expect a non-empty list of filenames, I think we should require the data module to be present in the bunch.